include/multiple_object_tracking/fusing.hpp

```diff
@@ -111,6 +111,7 @@ constexpr Visitor covariance_intersection_visitor{
 
     // Create a new fused track with updated state and covariance
     auto fused_track{track};
+    fused_track.timestamp = detection.timestamp;
```

Comment:
During track fusion, I know we do some temporal alignment, projecting detection data to some common future time. Should we be using this future time stamp or is that projection only for association and dropped after that?

Reply:
We are using it. Detections get propagated to the current system time before this function is called. `track.timestamp` is from the previous pipeline execution, but `detection.timestamp` is the current system time when the pipeline executes.